api/login_api.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import division
+from __future__ import print_function
+
+import logging
+
+from google.oauth2 import id_token
+from google.auth.transport import requests
+from flask import session
+
+
+from framework import basehandlers
+# from framework import permissions
+# from framework import ramcache
+# from internals import models
+
+class LoginAPI(basehandlers.APIHandler):
+  """Create a session using the id_token generated by Google Sign-In."""
+
+  def do_post(self):
+    token = self.get_param('id_token')
+    message = "Unable to Authenticate"
+
+    try:
+      CLIENT_ID = '77756740465-e5r4o15qg4hkdfiucjpl231o79k3ipjv.apps.googleusercontent.com'
+      idinfo = id_token.verify_oauth2_token(token, requests.Request(), CLIENT_ID)
+      # userid = idinfo['sub']
+      # email = idinfo['email']
+      session["id_token"] = token
```

Comment:
This makes sense, but it raises the question of how a developer who is working on chromestatus can log in a test users when running locally.  Do you have an idea of how to support that?

Reply:
I read on [StackOverFlow](https://stackoverflow.com/questions/17657879/does-google-provide-test-users-for-integration-testing) that Google Sign-In does not support test users. So, the developer will have to log in atleast once by using a Google Account. 

One idea ( but only if we HAVE to support test users) is we can create a separate login system only for the DEV environment.

Reply:
Let's see how it works out to use real accounts in the development environment.  We might not need the complexity of adding a dev-only login system.  And, there is always a risk that a dev-only login system might somehow be enabled in prod by some mistake in the future.

Reply:
Okay